### PR TITLE
TST: test suite was using old kwarg, swap to new

### LINKED ISSUE #295: Unit tests are failing on latest tag
Unit test outputs on travis have many instances of:
```
>           templated = template_main([project_filename], template=template_fp)
E           TypeError: main() got an unexpected keyword argument 'template'
```
sourced from https://github.com/pcdshub/pytmc/blob/master/pytmc/tests/test_commandline.py

It looks like we changed the `template` kwarg to `templates`: https://github.com/pcdshub/pytmc/blob/85fa86d9a716651aee70831e5bb8f69cc9566eed/pytmc/bin/template.py#L680
But I'm very confused because the master was passing with this in it

Failing tags:
https://app.travis-ci.com/github/pcdshub/pytmc/builds/254953822
https://app.travis-ci.com/github/pcdshub/pcds-envs/builds/255486106

Was passing on master prior to the tag (with failing docs...):
https://app.travis-ci.com/github/pcdshub/pytmc/builds/254263773

### PR DESCRIPTION
**Pull request by ZLLentz**
closes #295 probably
see linked issue for details

### CODE REVIEW

**Review by klauer — DISMISSED** [review]
This looks right to me :+1:
(Confused as to how we got here, but 🤷 )

Edit: Oh, I missed the 2nd comment here which likely explains it:
> On closer inspection, the master build was actually failing but was marked as a success (because coverage passed?)
> I think Ken fixed this CI issue recently but this PR came before that

Edit 2: I don't know my own code

Edit 3: I'll stop editing now

**Comment by ZLLentz** [comment]
More small edits needed: apparently the old interface wanted a file-handle-like object

**Comment by ZLLentz** [comment]
haha ok, specifically I need to give a string filepath, while the old tests were written assuming that passing a file-like object is ok

**Comment by ZLLentz** [comment]
Test were skipping for me because I forgot to clone with submodules, classic error

Hopefully this is all that's left because I have relearned that no test suite change is ever simple/easy/quick

**Comment by ZLLentz** [comment]
I'd appreciate a brief re-review @klauer since when you took a look it was only a two line change with two characters added

File: pytmc/tests/templates/basic_test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+{% for project in projects %}{{ project }}{% endfor %}
```
Comment (klauer) [inline pytmc/tests/templates/basic_test.txt:1]: I think we need a:
```
recursive-include pytmc/tests *.txt
```
In `MANIFEST.in`

**Review by klauer — APPROVED** [review]
Aside from the MANIFEST addition, I think this looks good :+1:
Thanks for cleaning up my mess!

File: pytmc/tests/templates/basic_test.txt (hunk 1 above)
Comment (ZLLentz) [reply pytmc/tests/templates/basic_test.txt:1]: correct, I forgot about this

**Pull request merged**